.babelrc

```diff
@@ -3,7 +3,6 @@
     "airbnb"
   ],
   "plugins": [
-    "add-module-exports",
```

Comment:
why must this be removed? this is a breaking change, and consumers should *never* see the `.default` of babel's interop.

I'd prefer to extract the pieces you need to separate files, each with their own `module.exports =` (via `export default`).

Reply:
I removed it because when there is a named export it automatically disables itself. This isn't a breaking change because I manually added `module.exports =` in the source file.

I'll update the PR to expose the different functions from different files.